### DISCUSSION TITLE
How to preserve the server settings to LTSP clients

### DISCUSSION THREAD

**Discussion by hc2021hc** [ltsp/ltsp#480] [Q&A]
Using Ubuntu LTS 20.04, I have set the LTSP server not to screen lock and never blank screen and run ltsp image /, however, in LTSP client, I found such settings were not set.  May I know how to have LTSP clients follow the same settings as LTSP server?

**Comment by alkisg** [comment]
Rephrase your question like this:

> I have a multiuser system. I updated the GNOME settings of **one** user. How can I force the same GNOME settings to **all** users?

I.e. this question isn't specific to LTSP. For GNOME, the answer is to use locked down dconf settings, you can read about it there:

https://wiki.gnome.org/Projects/dconf/SystemAdministrators

For other desktop environments, you'd search for similar documentation.
